src/Gente/World.js

```diff
@@ -25,9 +26,11 @@ export default class World {
 	takeTurn() {
 		this.incrementYear();
 
-		for (var i = 0; i < this.populace.length; i++) {
+    console.log('World population on this turn: %i', this.populace.length)
+    let sample = sampleSize(this.populace, this.settings.turnSampleSize)
```

Comment:
If you use a fixed sampleSize every turn then the percentages of people who age, die, marry, birth, etc gets lower and lower every turn. We'll have to come up with something else.